src/components/dropdown/constants.ts

```diff
@@ -7,3 +7,8 @@ export const OPEN_DROPDOWN_KEY_CODES = [
 ];
 
 export const CLOSE_DROPDOWN_KEY_CODES = [KeyCodes.ESCAPE_KEY_CODE, KeyCodes.TAB_KEY_CODE];
+
+export const EVENT_NAME = {
```

Comment:
Please use enum, it is customary to name it in the CamelCase:
export enum EventName {
  ON_KEY_DOWN = 'onKeyDown',
  ON_CLICK: 'onClick',
}